core/cc/linux/get_vulkan_proc_address.cpp

```diff
@@ -38,7 +38,7 @@ typedef size_t VkInstance;
 struct MesaLLVMOpener {
   MesaLLVMOpener() {
     char name[512];
-    for (int i = 3; i <= 9; i++) {
+    for (int i = 3; i <= 12; i++) {
```

Comment:
odd... i have memory of already having done this, since my personal desktop has had LLVM10 for a while now. Can you up the 12 to 20, so we don't have to deal with this again for a while?

Reply:
Sure, done.